python/ray/serve/schema.py

```diff
@@ -883,3 +888,26 @@ def serve_status_to_schema(serve_status: StatusOverview) -> ServeStatusSchema:
         app_status=serve_status.app_status,
         deployment_statuses=serve_status.deployment_statuses,
     )
+
+
+@PublicAPI(stability="alpha")
+@dataclass
+class DeploymentStatusOverview:
+    status: DeploymentStatus
+    message: str
+
+
+@PublicAPI(stability="alpha")
+@dataclass
+class ApplicationStatusOverview:
+    status: ApplicationStatus
+    message: str
+    last_deployed_time_s: float
+    deployments: Dict[str, DeploymentStatusOverview]
+
+
+@PublicAPI(stability="alpha")
+@dataclass(eq=True)
+class ServeStatus:
+    http_proxies: Dict[str, HTTPProxyStatus] = field(default_factory=dict)
```

Comment:
Since we're planning to unify HTTP and gRPC, what do you think about naming this `proxies` instead of `http_proxies`?

Reply:
Ah, good point! @GeneDer do you have insight on how to best structure the status response? One thing is though that [ServeInstanceDetails](https://github.com/ray-project/ray/blob/master/python/ray/serve/schema.py#L807) currently has a field called `http_proxies`. Would it be better to use a unified `proxies` or separate into something like `http_proxies` and `grpc_proxies`?

Reply:
yea, I have a follow up for renaming everything from httpxxxx to just xxxx. There will only be one proxy actor for both HTTP and gRPC and so only one statue. For external uses, I think it's better to expose just `proxies` like you mentioned. I can change all the internal naming after gRPC is up :)

Reply:
Sounds good, I've renamed `http_proxies` to `proxies`